tensorflow/python/ops/parsing_ops.py

```diff
@@ -884,6 +884,24 @@ def decode_raw(input_bytes,
         input_bytes, out_type, little_endian=little_endian, name=name)
 
 
+@tf_export("io.encode_raw", v1=[])
+@dispatch.add_dispatch_support
+def encode_raw(tensor, name=None):
+  """Convert tensor into raw byte strings.
+
```

Comment:
The documentation should include an example that illustrates how the API can be used. (Ideally, in the context of a realistic use).

Reply:
@jsimsa Sorry for this late reply. I'm not sure how to provide a realistic use of this api. From the origin issue #46493, it seems that the user hopes to use this api for tf-serving and that scenario would be too complex.

Reply:
If the documentation cannot explain what use case the API should be used for, then I am not sure it makes sense to introduce the API.

My suggestion for you would be to contact the authors of #46493 to understand how they would expect to use it and provide a high-level description, ideally with a (pseudo)code example that illustrates usage, in the documentation of the new API.